Resources/Prototypes/_RMC14/Entities/Mobs/Xeno/base_xeno.yml

```
@@ -78,6 +78,8 @@
     - Piercing
     - Heat
     - Shock
+  - type: CMDefibrillatorBlocked
+    popup: "Why would you want to bring this thing back!"
```

Likewise this should be changed to use the loc id
```suggestion
    popup: defibrillator-xeno
```